src/nuitka_tasks/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.0.1"
```

Comment:
I do not like assigning a version to something like this, but I get that it might be required.